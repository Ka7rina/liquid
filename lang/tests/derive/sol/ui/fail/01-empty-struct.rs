use liquid::{InOut, State};
use liquid_lang as liquid;

#[derive(InOut, State)]
pub struct MyStruct {}

fn main() {}
