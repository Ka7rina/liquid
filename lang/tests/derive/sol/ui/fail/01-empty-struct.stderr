error: empty structs are not supported
 --> $DIR/01-empty-struct.rs:5:21
  |
5 | pub struct MyStruct {}
  |                     ^^
